kotlin-analysis-api/testData/jvmName.kt

```diff
@@ -2,10 +2,25 @@
 // EXPECTED:
 // (getX, setX), (getY, null)
 // (getX, setX), (getY, null)
+// stringParameter
```

Comment:
Somehow this is still `stringParameter, stringParam` on my local checkout.

Reply:
Sorry, I pushed the debug version. Fixed it now.